core/script.lisp

```diff
@@ -377,7 +377,7 @@ script structure:
     <20-byte witness-program>"
   (and
    (segwit-p script-pubkey)
-   (eq 0 (command-number (aref (script-commands script-pubkey) 0)))
+   (= 0 (command-number (aref (script-commands script-pubkey) 0)))
```

Comment:
This was done for a purpose during debugging, but I don't remember why I kept it. Fine by me.